src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,26 @@
-export const CommentList = () => <>Put the list here</>;
+import comments from '../../api/comments.json';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export function CommentList({ postId }) {
+  const postHasComments = comments.some(comment => comment.postId === postId);
+  const targetComments = comments.filter(comment => comment.postId === postId);
+
+  return (
+    <>
+      {postHasComments ? (
+        <div className="CommentList">
+          {targetComments.map(comment => (
+            <CommentInfo commentId={comment.id} key={comment.id} />
+          ))}
+        </div>
```

Comment:
leave only it, and as you pass comments list already, so pass `coment` instead of commentId